### PR TITLE
ROX-18181: Add 4.0.3 patch info to RNs

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version(s):
- `rhacs-docs-4.0`

[Issue](https://issues.redhat.com/browse/ROX-18181)

Link to docs preview:

- https://file.rdu.redhat.com/kcarmich/ROX-18181-acs-rn-403/release_notes/40-release-notes.html
- specifically: https://file.rdu.redhat.com/kcarmich/ROX-18181-acs-rn-403/release_notes/40-release-notes.html#resolved-in-version-403

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by charmik-redhat — APPROVED** [review]
LGTM

File: release_notes/40-release-notes.adoc
Comment (skopacz1) [inline release_notes/40-release-notes.adoc:482]: Wording suggestion, including the use of the preferred term, "[update](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update)":

```suggestion
* Fixed an issue where updates failed because {product-title-short} applied PSPs, even when not enabled. (ROX-17771, ROX-17734)
```

**Review by skopacz1 — COMMENTED** [review]
Just left one comment, looks good! I am new to reviewing, so please wait until @adellape approves my review before implementing feedback. Thanks!

**Review by adellape — COMMENTED** [review]
@skopacz1 Just 1 more note, otherwise peer review comments LGTM 👍

File: release_notes/40-release-notes.adoc
Comment (kcarmichael08) [reply release_notes/40-release-notes.adoc:482]: ACS is a little weird because the existing docs use "upgrade," so to use "update" in this context would be inconsistent...but it's a good point that we should eventually change our language to match the standards.

File: release_notes/40-release-notes.adoc
Comment (kcarmichael08) [reply release_notes/40-release-notes.adoc:481]: It looks like in ACS we have been using "Pod Security Policies (PSPs)" (no backtick) to refer to PSPs. This seems consistent with Red Hat and Kubernetes docs.

**Pull request merged**
